DeepSpeech.py

```diff
@@ -761,14 +761,17 @@ def calculate_report(results_tuple):
     samples = []
     items = list(zip(*results_tuple))
     mean_wer = 0.0
+    total_label_length = 0.0
     for label, decoding, distance, loss in items:
         sample_wer = wer(label, decoding)
```

Comment:
Why not retrieving the edit-distance directly here (calling `levenshtein()`)? I think this is nicer than re-multiplying the word count later to aggregate it (`mean_wer += sample_wer * label_length`).